src/DynamoCoreWpf/Utilities/CrashUtilities.cs

```diff
@@ -1,25 +1,30 @@
-﻿using Dynamo.Configuration;
+using Dynamo.Configuration;
 using Dynamo.PackageManager;
 using Dynamo.Utilities;
 using System;
-using System.Collections.Generic;
 using System.Linq;
 
 namespace Dynamo.Wpf.Utilities
 {
     static class CrashUtilities
     {
-        internal static string GithubNewIssueUrlFromCrashContent(object crashContent)
+        internal enum ReportType
+        {
+            Bug,
+            Crash
+        }
+
+        internal static string GithubNewIssueUrlFromCrashContent(object crashContent, ReportType reportType = ReportType.Crash)
         {
             var baseUri = new UriBuilder(Configurations.GitHubBugReportingLink);
 
-            var issueTitle = "Crash report from Dynamo {0}";
+            var issueTitle = "{0} report from Dynamo {1}";
             var dynamoVersion = AssemblyHelper.GetDynamoVersion().ToString() ?? "2.1.0+";
 
             // append the title and body to the URL as query parameters
             // making sure we properly escape content since stack traces may contain characters not suitable
             // for use in URLs
-            var title = "title=" + Uri.EscapeDataString(string.Format(issueTitle, dynamoVersion));
+            var title = "title=" + Uri.EscapeDataString(string.Format(issueTitle, reportType, dynamoVersion));
```

Comment:
It's been a while, can you double check here if it would be number or string. I remember vaguely that the real value of Enums will be number and you may need to grab the name of the Enum specifically here.

Reply:
It appears to pull the string from the enum on the string formatting portion.

Reply:
Thank you for checking @johnpierson maybe the string.Format took care of it underneath already.